data/map systems.txt

```diff
@@ -24059,6 +25463,45 @@ system Nocte
 		distance 2260.97
 		period 3701.13
 
+system Noitakt
+	pos -129.13889 225.41667
+	government Syndicate
+	link Mirach
+	link Schedar
+	asteroids "small rock" 4 2.604
+	asteroids "large rock" 7 3.069
+	asteroids "small metal" 9 3.069
+	asteroids "medium metal" 9 4.1385
+	asteroids "large metal" 2 4.278
+	minables aluminum 1 5.7519675
+	minables lead 7 6.190245
+	haze _menu/haze-133
+	fleet "Large Core Pirates" 800
+	fleet "Small Core Pirates" 1200
+	fleet "Small Core Merchants" 600
```

Comment:
```suggestion
	fleet "Small Core Merchants" 950
	fleet "Large Core Merchants" 1100
	fleet "Small Syndicate" 3000
	fleet "Large Syndicate" 4500
	fleet "Small Core Pirates" 2500
	fleet "Large Core Pirates" 4000
```
Based on the fleet frequencies in Mirach and Schedar, since those are the only systems this is connected to.